assets/additionalAttributions.txt

```diff
@@ -62,7 +62,7 @@ All rights reserved.
 
 Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
The PackageManagement license has changed.  Can you update to this https://github.com/OneGet/oneget/blob/WIP/LICENSE 

Otherwise, revert this change and file an issue, mentioning me.

Reply:
@RDIL 	has a PR open #8676

Reply:
Resolved in #8676